docs/sql-programming-guide.md

```diff
@@ -790,6 +790,15 @@ Configuration of Parquet can be done using the `setConf` method on `SparkSession
     </p>
   </td>
 </tr>
+<tr>
```

Comment:
PS this does not look like the right place for this option. This section covers Parquet. There's a later "other options" section covering broadcast join. Move it there.